src/DependencyInjection/Configuration.php

```diff
@@ -65,6 +66,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->append($this->decorateDriverConfig())
                 ->append($this->decorateSessionConfig())
                 ->append($this->decorateTransactionConfig())
+                ->scalarNode('log_level')
```

Comment:
I'm a little bit confused by this. You mean only errors will be logged or everything it logs will be logged as an error? @exaby73

Reply:
It will be logged according to this logic:

https://github.com/neo4j-php/neo4j-php-client/blob/5bf29c7e4bddf55130e75d0b75cfc14e19379cfd/src/Common/Neo4jLogger.php#L67-L70

Reply:
hmm, in the code it uses the term `shouldLog` which is a lot clearer. Maybe we should call it: `min_log_level` with the description: `the minimum severity the driver will log`